AdminUi/apps/admin_ui/lib/pages/login_page.dart

```diff
@@ -34,56 +38,128 @@ class _LoginScreenState extends State<LoginScreen> {
 
   @override
   Widget build(BuildContext context) {
+    final screenSize = MediaQuery.of(context).size;
+    final topPadding = screenSize.height > 1080 ? screenSize.height * 0.2 : screenSize.height * 0.3;
+
     return Scaffold(
       appBar: AppBar(
-        title: const Text('Enmeshed Admin UI - Login'),
+        title: Row(
+          mainAxisSize: MainAxisSize.min,
+          children: [
+            SvgPicture.asset(
+              'assets/logo.svg',
+              width: 30,
+              height: 30,
+            ),
+            const SizedBox(width: 10),
+            const Text(
+              'enmeshed',
+              style: TextStyle(fontWeight: FontWeight.bold),
+            ),
+            const SizedBox(width: 10),
+            const Text('Admin UI'),
+          ],
+        ),
+        leading: const SizedBox(
+          width: 40,
+        ),
       ),
-      body: Center(
-        child: SizedBox(
-          width: 300,
-          child: Column(
-            mainAxisSize: MainAxisSize.min,
-            children: [
-              SvgPicture.asset('assets/logo.svg', width: 200, height: 200),
-              const SizedBox(height: 40),
-              TextField(
-                controller: _apiKeyController,
-                focusNode: _apiKeyFocusNode,
-                decoration: const InputDecoration(
-                  labelText: 'API Key',
-                  border: OutlineInputBorder(),
+      body: SingleChildScrollView(
+        child: Padding(
+          padding: EdgeInsets.only(top: topPadding),
+          child: Align(
+            alignment: Alignment.topCenter,
+            child: SizedBox(
+              width: 500,
+              height: 250,
+              child: Card(
+                child: SizedBox(
+                  width: 300,
+                  child: Padding(
+                    padding: const EdgeInsets.only(top: 20, left: 25, right: 25),
```

Comment:
Same as for Button and TextField: this should probably be some global style.

Reply:
Extracted into a global style.